e2e/cmd/run_tests/01_provision_client/main.go

```diff
@@ -190,7 +190,7 @@ func action(ctx context.Context, cmd *command.Command) error {
 
 	// TODO: remove this once the packages installed below are MIRed and installed by default with adsys
 	log.Infof("Installing universe packages required for some policy managers...")
-	_, err = client.Run(ctx, "DEBIAN_FRONTEND=noninteractive apt-get install -y ubuntu-proxy-manager python3-cepces")
+	_, _ = client.Run(ctx, "DEBIAN_FRONTEND=noninteractive apt-get install -y ubuntu-proxy-manager python3-cepces")
```

Comment:
hum, shouldn’t we fail quickly if we can’t install?

Reply:
shouldn’t we log them at least, so that if some policy fails where we expected to have them working, we can see that was because of the package failing to install for $reasons?

Reply:
Ah, that’s perfect then! :)